src/game/entities/player.ts

```diff
@@ -265,3 +265,14 @@ export class Player extends Entity {
     }
   }
 }
+
+const calculateMaxHPFromLevel = (level: number): number =>
+  10 * Math.floor((10 * level ** 1.3 + 90) / 10);
+
+const calculateDmgFromLevel = (level: number): number => 9 + sumToOneSqrt(level);
+const calculateArmFromLevel = (level: number): number => 9 + sumToOneSqrt(level);
+
+const sumToOneSqrt = (x: number): number => {
+  if (x === 1) return 1;
+  return Math.round(x ** 0.5) + sumToOneSqrt(x - 1);
```

Comment:
What made you decide to use a really expensive operation for this?
Admittedly on level up it isn't too bad, as it is just adding √level, but why?